src/pages/orgadmin/ExamInformation.tsx

```
@@ -0,0 +1,189 @@
+
+const rangeConfig = {
+    rules: [
+        { type: 'array' as const, required: true, message: 'Please select time!' },
+    ],
+};
+
+type ExamInformationProps = {
+    onFinishFun: (values: ExamRequest) => void;
+};
+
+const ExamInformation: React.FC<ExamInformationProps> = ({ }) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)